src/app/blocks/load-schema-block/load-schema-block.component.ts

```diff
@@ -32,15 +32,16 @@ export class LoadSchemaBlockComponent  extends BaseBlockComponent {
     const baseSchema = (!!this.schemaGetter)
       ? mappingUtility({ context: this.context, data: this.model }, this.schemaGetter)
       : this.schema;
-
-    const schemaDefinitions = {};
-    schemaDefinitions[baseSchema] = await this.resolveSchema(schemaDefinitions, baseSchema);
-    const jsonSchema = {
-      definitions: schemaDefinitions,
-      '$ref': `#/definitions/${baseSchema}`
-    };
-    // TODO: some fields may need uiSchema (eg widget overrides)
-    this.output.emit({ jsonSchema, uiSchema: {} });
+    if((typeof baseSchema === 'string') && baseSchema.length > 0) {
```

Comment:
This could be more symetrical, e.g:
```suggestion
    if((typeof baseSchema === 'string') && (baseSchema.length > 0)) {
```

Reply:
Or maybe just the outer brackets.

Reply:
done.